pythoncode/calc.py

```diff
@@ -0,0 +1,19 @@
+# Perimeter of Square
+def calculatePerimeter(l):
+    return 4*l
+
+# Area of Square
+def calculateArea(l):
+    return l*l
+
+
+expression = input("Type a function: ")
+
+for l in range(1, 5):
+    if (expression == 'calculatePerimeter(l)'):
+        print("If length is ", l, ", Perimeter = ", eval(expression))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you do not have absolute control over the contents of the string passed to the `eval()` function, they might be maliciously used to execute an arbitrary function such as `__import__(&#39;os&#39;).system(&#34;rm -rf $HOME&#34;)`. If the string is expected to only contain literals, consider using the `ast.literal_eval()` method to safely evaluate strings containing Python values from untrusted sources.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-use-strings-to-call-functions-methods)

Similar issue at line number 16.